src/LocalTranslationsBatcher.php

```diff
@@ -151,7 +164,30 @@ protected function getExtensionsToTranslate(): array {
       $extensions[] = $name;
     }
 
-    return $extensions;
+    return array_intersect_key(locale_translation_project_list(), array_combine($extensions, $extensions));
```

Comment:
Let's not do this here but inside the `getExtensionsToTranslate()` simply return the extension objects we need (which contain the .info data you need). Also, see my other comment about where to call this. probably from inside the operation in the first batch run.

Reply:
Unfortunately extension object doesn't contain information about location of *.po files folder (`interface translation server pattern`).
This value is dynamic and provides information through implementation of `hook_locale_translation_projects_alter` hook.